src/toil/batchSystems/abstractBatchSystem.py

```diff
@@ -160,16 +162,17 @@ def supportsWorkerCleanup():
 
     @staticmethod
     def workerCleanup(workerCleanupInfo):
-        '''
+        """
         Cleans up the worker node on batch system shutdown. For now it does nothing.
         :param collections.namedtuple workerCleanupInfo: A named tuple consisting of all the
         relevant information for cleaning up the worker.
-        '''
+        """
         assert workerCleanupInfo.__class__.__name__ == 'WorkerCleanupInfo'
         workflowDir = getToilWorkflowDir(workerCleanupInfo.workflowID, workerCleanupInfo.workDir)
-        # TODO: logic here for cleanup flag
-        shutil.rmtree(workflowDir)
-
+        dirIsEmpty = os.listdir(workflowDir) == []
```

Comment:
@hannes-ucsc I checked and did not find a method that simplifies checking if a directory is empty